src/datepicker/datepicker.component.ts

```diff
@@ -454,6 +458,38 @@ export class DatePicker implements
 		});
 	}
 
+	/**
+	 * Handles the initialization of event listeners for the datepicker input and range input fields.
+	 */
+	protected addInputListeners() {
+		if (this.isFlatpickrLoaded()) {
+			// Allows focus transition from the datepicker input or range field to
+			// flatpickr calendar using a keyboard.
+			const addFocusCalendarListener = (element: HTMLInputElement) => {
+				element.addEventListener("keydown", (event: KeyboardEvent) => {
+					if (event.key === "ArrowDown") {
+						if (!this.flatpickrInstance.isOpen) {
+							this.flatpickrInstance.open();
+						}
+
+						(
```

Comment:
this looks... 🤔 
If `this.flatpickrInstance.calendarContainer` is falsey, `focus` will throw?
So how about we assign the value to a var, check if it's falsey and focus in the if body?
That should also make this more readable.